internal/uvm/network.go

```diff
@@ -124,13 +124,22 @@ func (uvm *UtilityVM) NCProxyEnabled() bool {
 	return uvm.ncProxyClientAddress != ""
 }
 
-func (uvm *UtilityVM) CreateNCProxyClient() (ncproxyttrpc.NetworkConfigProxyService, error) {
+type ncproxyClient struct {
+	raw *ttrpc.Client
+	ncproxyttrpc.NetworkConfigProxyService
+}
+
+func (n *ncproxyClient) close() error {
```

Comment:
Recommend exporting this as `Close` so we can use `*ncproxyClient` as an `io.Closer` if we want to.